src/components/Article.jsx

```diff
@@ -49,7 +49,7 @@ const Article = () => {
               <Rating
                 onRate={articleRating}
                 data-cy='article-rating-button'
-                defaultRating={setRating(article.rating)}
+                rating={setRating(article.rating)}
                 maxRating={5}
                 icon='star'
```

Comment:
Should we just set `rating = {article.rating}`

Reply:
That fixed the problem with the page refresh, because the stars were disappearing when refreshing the page.

Reply:
yes, now you don't see stars when user rates it, but it should be a condition we dissected, if rated - `setRating(user.rating)` else `setRating(article.rating)`

Reply:
And `user.rating` would then be from this secondary request we talked about? :P